lms/static/scripts/frontend_apps/components/test/ErrorDisplay-test.js

```diff
@@ -159,12 +174,11 @@ describe('ErrorDisplay', () => {
     'should pass a11y checks',
     checkAccessibility({
       // eslint-disable-next-line react/display-name
-      content: () => (
-        <ErrorDisplay
-          message="Something went wrong"
-          error={new Error('Oh no')}
-        />
-      ),
+      content: () =>
+        renderErrorDisplay({
+          description: 'Something went wrong',
+          error: Error('Oh no'),
+        }),
```

Comment:
This was not using an enzyme wrapper (`mount(<ErrorDisplay />)`) but the component itself `<ErrorDisplay />`, so it could be this test fails now.

Reply:
It still passes. This function usually works with both an enzyme wrapper and a DOM node. However, it could fail in the first case if the component renders more than one node internally.

It is not the case here, so both work.